tests/Unit/Services/Contact/Gift/DestroyGiftTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Tests\Unit\Services\Contact\Conversation;
+
+use Tests\TestCase;
+use App\Models\Contact\Gift;
+use App\Models\Account\Account;
+use App\Services\Contact\Gift\DestroyGift;
+use Illuminate\Validation\ValidationException;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class DestroyGiftTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    public function test_it_destroys_a_gift()
+    {
+        $gift = factory(Gift::class)->create();
+
+        $this->assertDatabaseHas('gifts', [
```

Comment:
This test here is a bit useless. You are testing if the factory works, which is not something you should test. But I understand that you want to make sure that the database really contains something.... it’s philosophical.

Reply:
That's it. I want to test the object was here, and then really been destroyed.